### PR TITLE
Feature/nav search bar crw 106

### PR DESCRIPTION
**Pull request by AbdallaEmadEldinAbdalla**
[CRW-106](https://techxarya.atlassian.net/browse/CRW-106)

pref: Implement asset search inside nav-bar.

[CRW-106]: https://techxarya.atlassian.net/browse/CRW-106?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/shared/assetDropdown/index.tsx [hunk 1]
```diff
@@ -71,41 +93,41 @@ export const AssetDropdown = (props: AssetDropdownTypes) => {
...
                     }
                 </DropdownMenu.Trigger>
 
                 <DropdownMenu.Portal className="z-10">
-                    <DropdownMenu.Content className="min-w-[400px] bg-grey-2 rounded-md max-h-[340px] overflow-scroll" sideOffset={15}>
-                        <Col className="gap-4 p-4 mb-3 absolute bg-grey-2 w-full rounded-lg">
-                            <h3 className="font-extrabold text-white text-xl">{props.t('selectAsset')}</h3>
+                    <DropdownMenu.Content className={clsx({ "min-w-[400px] max-h-[340px]": !showTopCoinsList, "min-w-[500px] max-h-[440px]": showTopCoinsList }, "bg-grey-2 rounded-md overflow-scroll")} sideOffset={sideOffset} align={align} side={side}>
+                        <Col className={clsx({ "p-4": showContentHeaderLabel, "p-2": !showContentHeaderLabel }, "gap-4 mb-3 absolute bg-grey-2 w-full rounded-lg z-50")}>
+                            {showContentHeaderLabel && <h3 className="font-extrabold text-white text-xl">{contentHeaderLabel || t('selectAsset')}</h3>}
```
Comment (almujtabamohammed) [inline src/components/shared/assetDropdown/index.tsx:110]: Fix translation for "selectAsset"

**Review by almujtabamohammed — CHANGES_REQUESTED** [review]
Fix responsiveness

**Comment by AbdallaEmadEldinAbdalla** [comment]
> Fix responsiveness

@almujtabamohammed  i don't have any design for the mobile view.

**Comment by almujtabamohammed** [comment]
> > Fix responsiveness
> 
> @almujtabamohammed i don't have any design for the mobile view.

**Comment by almujtabamohammed** [comment]
> > Fix responsiveness
> 
> @almujtabamohammed i don't have any design for the mobile view.

Can you check with Daryl and Alex, because it's not usable right now on mobile

**Review by almujtabamohammed — APPROVED** [review]

**Pull request merged**